test/api/controllers/adminController.test.js

```diff
@@ -99,41 +56,46 @@ describe('Test: admin controller', function () {
 
     it('should add the new collection to the cache', () => {
       this.timeout(50);
+      sandbox.stub(kuzzle.workerListener, 'add').resolves({});
+      sandbox.spy(kuzzle.indexCache, 'add');
```

Comment:
As far as I understand how sinon.js works, I wonder if we should rather use always "stub" instead of "spy":
In a unit-test, we don't really want that the method kuzzle.indexCache.add is actually executed... we just want to check that its is called by the updateMapping() method.

Using only "stubs", we ensure that only the code we want to test is actually executed.